server/channels/api4/report.go

```diff
@@ -21,7 +21,7 @@ func (api *API) InitReports() {
 }
 
 func getUsersForReporting(c *Context, w http.ResponseWriter, r *http.Request) {
-	if !(c.IsSystemAdmin()) {
+	if !c.App.SessionHasPermissionTo(*c.AppContext.Session(), model.PermissionSysconsoleReadUserManagementUsers) {
```

Comment:
Do we have any such check in the client as well? can we cross check

Reply:
No, but we have a ticket for addressing the client issue more broadly - https://mattermost.atlassian.net/browse/MM-59311

Reply:
and its assigned to me 🤦🏽  😆